src/translation/tr.json

```diff
@@ -0,0 +1,1760 @@
+{"en_US":{
```

Comment:
I think this should be `"en-US"` instead? At least it does not work on my laptop this way.